ReportSimulationOutput/measure.rb

```diff
@@ -304,9 +309,23 @@ def energyPlusOutputRequests(runner, user_arguments)
       end
     end
     if has_electricity_production
-      result << OpenStudio::IdfObject.load('Output:Meter,ElectricityProduced:Facility,runperiod;').get # Used for error checking
+      result << OpenStudio::IdfObject.load('Output:Meter,Photovoltaic:ElectricityProduced,runperiod;').get # Used for error checking
+      result << OpenStudio::IdfObject.load('Output:Meter,Cogeneration:ElectricityProduced,runperiod;').get # Used for error checking
+      result << OpenStudio::IdfObject.load('Output:Meter,Photovoltaic:Inverter,runperiod;').get # Used for error checking
+      result << OpenStudio::IdfObject.load('Meter:Custom,Photovoltaic:Inverter,electricity,*,Inverter Conversion Loss Decrement Energy;').get
```

Comment:
Help me understand this... We are replacing `ElectricityProduced:Facility` with a subset of its components, is that right? This has all gotten much more complicated so some comments in the code would be nice.